webapp/src/client.ts

```diff
@@ -337,6 +338,11 @@ export default class CallsClient extends EventEmitter {
 
         this.closed = true;
         if (this.peer) {
+            this.getStats().then((stats) => {
```

Comment:
Well, if the call drops before connecting then those stats are of no use anyway since they are about media tracks. Once media is flowing a drop should always go through this handler unless there's a leak but that would be a bug.